src/index.scss

```diff
@@ -12,6 +12,7 @@ html {
 }
 
 body {
+  height: 100vh;
```

Comment:
@dalefukami - this fixes the hidden dropdown. Brett and I had a session and looked into it some more and appears Radix is forcing an `overflow: hidden !important` to the body when the dropdown menu is open (likely to prevent scrolling when open). To resolve, adding `100vh` to the body here ensures the dropdown menu is visible once opened. I wasn't able to see any unwanted consequences of this, but let me know if there is causes problems elsewhere.

Reply:
oh man, that sucks. Don't like it when a 3rd party is forcing styles on elements it doesn't own. :grimacing: